scripts/jenkins/Makefile.jenkins

```diff
@@ -485,3 +485,11 @@ check-leaks:
 		echo "Credentials leaked! Please check $(PWD)/leak-check.out"; \
 		exit 1; \
 	fi
+
+h2o-docker-build:
+	docker build -t $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) -f docker/public/Dockerfile-h2o-release .
+
+h2o-docker-push:
+	docker login -u $(DOCKERHUB_USERNAME) -p $(DOCKERHUB_PASSWORD)
+	docker push $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)
```

Comment:
should we also update latest?

Reply:
I'm not sure what should be latest. I'd say latest is not just anything that came out or nightly. From my POV, it should be latest stable release. WDYT ?

Reply:
yes, latest should be stable, we can have also latest-nightly tag

Reply:
latest should correspond with what is on the download page

Reply:
That is any latest release, doesn't matter if from master or from rel- branch - or ?

Reply:
I think nightly is not really a release. So I would stick to stable, or we can switch the tags to latest and latest-stable, WDYT @michalkurka ?

Reply:
I wouldn't worry about the nightly builds at all. latest should correspond point to latest stable. nightly builds are sometimes misunderstood, having them tagged as "latest" could add to the confusion